src/MutantObservers/JsonLoggingObserver.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
```

Comment:
File level copyright block and license (do add your author name wherever preferred!)